mail/models.py

```diff
@@ -0,0 +1,19 @@
+"""
+Models for mail
+"""
+from django.contrib.auth.models import User
+from django.db import models
+
+from financialaid.models import TimestampedModel, FinancialAid
+
+
+class FinancialAidEmailAudit(TimestampedModel):
+    """
+    Audit table for the Financial Aid
+    """
+    acting_user = models.ForeignKey(User, null=False)
+    financial_aid = models.ForeignKey(FinancialAid, null=True, on_delete=models.SET_NULL)
```

Comment:
We thought it would be important to allow it to be null in case there happens to be a case where the `FinancialAid` object is deleted.  The last thing we'd want is for an audit record to be deleted.